### PR TITLE
Ypersky multi clones used capacity enhancement

### PR DESCRIPTION
**Pull request by ypersky1980**
This PR contains the following enhancements: 

1) Free ceph capacity calculation is added and the PVC/clones sizes are calculated by diving the free capacity be the required number of maximal PVCs in the cluster ( instead of the total capacity) 

2) Teardown problems should be resolved  

3) Added retries to overcome network glitches during which the test is unable to read data from logs.

### CODE REVIEW

File: ocs_ci/ocs/cluster.py [hunk 1]
```diff
@@ -641,6 +641,28 @@ def get_ceph_capacity(self):
...
+            logger.info(f"Number of replica : {replica}")
+            ct_pod = pod.get_ceph_tools_pod()
+            output = ct_pod.exec_ceph_cmd(ceph_cmd="ceph df")
+            total_avail = output.get("stats").get("total_bytes")
+            total_used = output.get("stats").get("total_used_raw_bytes")
+            total_free = total_avail - total_used
+            return total_free / replica / constants.BYTES_IN_GB
+        else:
+            # if the replica number is 0, usable capacity can not be calculate
+            # so, return 0 as usable capacity.
```
Comment (ramkiperiy) [inline ocs_ci/ocs/cluster.py:663]: replica 0 doesn't exist, we can remove this comment or even remove the else IMO.

**Comment by ypersky1980** [comment]
The fix is to increase teardown timeout to 3600 sec.

File: ocs_ci/ocs/cluster.py (hunk 1 above)
Comment (ypersky1980) [reply ocs_ci/ocs/cluster.py]: Removed it, also the "if - else".

**Review by ramkiperiy — APPROVED** [review]

**Review by pintojoy — APPROVED** [review]
LGTM

**Pull request merged**